index.js

```diff
@@ -95,8 +96,16 @@ Wemo.prototype._handleRequest = function(req, res) {
 };
 
 Wemo.prototype.getCallbackURL = function() {
+  var self = this;
   var getLocalInterfaceAddress = function() {
     var interfaces = os.networkInterfaces();
+    if (self._callbackInterface) {
```

Comment:
Actually the server still listens to all network interfaces ([line 69](https://github.com/timonreinhard/wemo-client/blob/master/index.js#L69)), this method just about constructing the callback URL. Would be great if you can change it so that `this._server.listen` takes the new option into account as well. (See https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback for docs.)

Let me know, if that's ok for you.